nixos/modules/services/web-apps/movim.nix

```diff
@@ -179,6 +179,17 @@ let
       "mysql" = "mysql.service";
     }
     .${cfg.database.type};
+
+  # https://github.com/movim/movim/issues/314
```

Comment:
nit: I don't actually see this CSP on the attached page. The link just says that movim doesn't support a "more restrictive" CSP policy. Is the CSP we have here a "less restrictive" CSP policy? Where did this CSP come from?

edit: After reading further, I see the quotes came from the original nginx configuration. IMO, all the more reason to separate the true *value* from the mechanism for injecting it into the relevant config files.

Reply:
> I think the ticket is worded improperly

Perhaps the best improvement here would be for you to leave a comment asking/clarifying this on that ticket?

Reply:
Thank you!